docs/content/ingestion/index.md

```diff
@@ -100,8 +100,9 @@ This is for realtime ingestion. Make sure to include "io.druid.extensions:druid-
 | type | String | This should say `avro_stream`. | no |
 | avroBytesDecoder | JSON Object | Specifies how to decode bytes to Avro record. | yes |
 | parseSpec | JSON Object | Specifies the format of the data. | yes |
+| schemaMappings | JSON Object | Specifies a mapping from field names found within the parseSpec to data within a complex avro structure. | no |
```

Comment:
Would it be more straight just using the _field_/_array_/_map_ grammer in the `dimensions`/`metricsSpec`? When getting the value of dimension/metric, first treat the name as simpleFieldAccessor, if got null then treat it as schema-mapping?